homeassistant/components/wolflink/.translations/sensor.pl.json

```diff
@@ -0,0 +1,85 @@
+{
+  "state": {
```

Comment:
We have reworked our translations design. There are no custom device classes implemented for the sensor entities that we want to translate states for. That's a requirement for the translation to work. The json data should be in a file named `strings.sensor.json`.

https://developers.home-assistant.io/docs/internationalization/core#stringssensorjson

Reply:
Hmm. I think I fixed that. TBH translations are a bit weird for me.